includes/class-graduate-degree-degree-type-taxonomy.php

```diff
@@ -71,4 +89,123 @@ public function register_taxonomy() {
+		foreach ( $this->term_meta_keys as $key => $args ) {
+			$args['show_in_rest'] = true;
+			$args['single'] = true;
+
+			register_meta( 'term', $key, $args );
+		}
+	}
+
+	/**
+	 * Saves the additional form fields whenever a term is updated.
+	 *
+	 * @since 0.8.0
+	 *
+	 * @param int $term_id The ID of the term being edited.
+	 */
+	public function save_term_form_fields( $term_id ) {
+		global $wp_list_table;
+
+		if ( 'editedtag' !== $wp_list_table->current_action() ) {
+			return;
+		}
+
+		// Reuse the default nonce that is checked in `edit-tags.php`.
+		check_admin_referer( 'update-tag_' . $term_id );
+
+		$keys = get_registered_meta_keys( 'term' );
+
+		foreach ( $this->term_meta_keys as $key => $meta ) {
+			if ( isset( $_POST[ $key ] ) && isset( $keys[ $key ] ) && isset( $keys[ $key ]['sanitize_callback'] ) ) {
+				// Each piece of meta is registered with sanitization.
+				update_term_meta( $term_id, $key, $_POST[ $key ] );
+			}
+		}
+
+		return;
+	}
+
+	/**
+	 * Retrieves all of the expected degree type meta assigned to a term.
+	 *
+	 * @since 0.8.0
+	 *
+	 * @param int $term_id
+	 *
+	 * @return array
+	 */
+	public static function get_all_term_meta( $term_id ) {
+		$term_meta = get_registered_metadata( 'term', $term_id );
+
+		return $term_meta;
+	}
+
+	/**
+	 * Captures information about a degree type as term meta.
+	 *
+	 * @since 0.8.0
+	 *
+	 * @param WP_Term $term
+	 */
+	public function term_edit_form_fields( $term ) {
+		$term_meta = self::get_all_term_meta( $term->term_id );
+
+		foreach( $this->term_meta_keys as $key => $meta ) {
+			if ( 'gs_degree_type_classification' !== $key ) {
+				continue;
+			}
+
+			if ( isset( $term_meta[ $key ][0] ) ) {
+				$selected_value = $term_meta[ $key ][0];
+			} else {
+				$selected_value = 'Other';
+			}
+
+			?>
+			<tr class="form-field">
+				<th scope="row">
+					<label for="<?php echo esc_attr( $key ); ?>"><?php echo esc_html( $meta['description'] ); ?></label>
+				</th>
+				<td>
+					<select name="<?php echo esc_attr( $key ); ?>" id="<?php echo esc_attr( $key ); ?>">
+						<option value="Other" <?php selected( 'None', $selected_value ); ?>>Other</option>
+						<option value="Doctorate" <?php selected( 'Doctorate', $selected_value ); ?>>Doctorate</option>
+						<option value="Masters" <?php selected( 'Masters', $selected_value ); ?>>Masters</option>
+						<option value="Graduate Certificate" <?php selected( 'Graduate Certificate', $selected_value ); ?>>Graduate Certificate</option>
+					</select>
+				</td>
+			</tr>
+			<?php
+		}
+	}
+
+	/**
+	 * Sanitizes a classification for a degree type.
+	 *
+	 * @since 0.8.0
+	 *
+	 * @param string $classification
+	 *
+	 * @return string
+	 */
+	public static function sanitize_degree_classification( $classification ) {
+		if ( empty( $classification ) ) {
+			$classification = 'Other';
+		}
+
+		if ( ! in_array( $classification, array( 'None', 'Masters', 'Doctorate', 'Graduate Certificate', 'Other' ), true ) ) {
```

Comment:
It looks like `None` would never be passed as a value, so it could be dropped from this array.